client/src/templates/Challenges/classic/show.tsx

```diff
@@ -377,7 +377,7 @@ function ShowClassic({
           containerRef={containerRef}
           description={description}
           // Try to remove unknown
-          editorRef={editorRef}
+          {...(editorRef && { editorRef: editorRef })}
```

Comment:
As far as I can see, change does nothing we want. `editorRef` cannot be `undefined`, so there's no point checking for that.
```suggestion
          editorRef={editorRef}
```